torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -264,40 +222,40 @@ bool validateKernelArg(
 }
 
 void validateKernelArgs(
-    const CudaKernel& entry,
+    CudaKernel* entry,
```

Comment:
@naoyam I'm removing the `const` here as we don't have the right const model on `FusionGuard`
Feel free to yell at me if there's a better way to do this.

Reply:
Sounds reasonable to me.